relay-kafka/src/config.rs

```diff
@@ -134,7 +132,7 @@ impl Default for TopicAssignments {
             sessions: "ingest-sessions".to_owned().into(),
             metrics: "ingest-metrics".to_owned().into(),
             metrics_sessions: None,
-            metrics_transactions: None,
+            metrics_transactions: "ingest-performance-metrics".to_owned().into(),
```

Comment:
This kind of makes sense. But I still would prefer that we do not set this up by default, and just provide the correct config. 
Since `Sentry` backend have such default already, I guess it's fine to have those too.

Reply:
Confirmed offline that session and transaction metrics should never go into the same topic these days, not even in self hosted or dev environments. Therefore, I think we have to change the default to get a working out-of-the-box experience.

Reply:
Will add a changelog entry though, in case anyone encounters problems with this change.